### ISSUE TITLE
Docs : update Homepage Documentation

### ISSUE THREAD

**Issue by Tayebsed93** [Orange-OpenSource/ouds-flutter#135]
Update the flutter documentation homepage to have something consistent with the other platforms.
Need design & web team help.

Change the style of dart doc to be conform to orange brand like for [android](https://android.unified-design-system.orange.com/) and [ios](https://ios.unified-design-system.orange.com/documentation/oudsthemescontract/) documentation

**Link Issue :** 
- [x] #239

**Comment by B3nz01d** [comment]
@nouha06 here are my feedbacks:

- [x] * can the side menu be updated to be in lowercase and have the top "Core" text says "Core Components"?
- [x] * the orange logo as the header should allow to go back to the home
- [x] * the sub title "OUDS" in the home page should be detailed to "Orange Unifed Design System"
- [x] * the token version should state "OUDS core token version: 1.9.0"

**Comment by nouha06** [comment]
@B3nz01d for the 3rd point, you mean this sub title ?

> OUDS Core Flutter provides Orange components to developers

**Comment by B3nz01d** [comment]
<img width="777" height="478" alt="Image" src="https://github.com/user-attachments/assets/490e3297-6a8b-4843-aca0-09cd658da33f" />
I'm talking about the title in the home page "OUDS"

**Comment by B3nz01d** [comment]
@nouha06 the side menu is now in full lowercase, could it be in capital letter on the irts letter of the component:

- [x] "Radio button"

with the exception of "PIN code input"

- [x] the sentence "OUDS stands for Orange Unified Design System."

should be reversed to "Orange Unified Design System is abbreviated to OUDS."

**Comment by B3nz01d** [comment]
Also on mobile the menu in not available in light mode and the following title overlaps on one line
![image](https://github.com/user-attachments/assets/64750935-9b91-4671-ab4a-e8e2ab5013f7)

**Comment by Tayebsed93** [comment]
Following our discussion with @B3nz01d, the requested changes have been approved. We have therefore merged the updates, as you mentioned

**Comment by B3nz01d** [comment]
Finished during sprint # 25

**Comment by B3nz01d** [comment]
Finished during sprint # 25
